lib/rabbitmq/cli/ctl/commands/quorum_status_command.ex

```diff
@@ -0,0 +1,56 @@
+## The contents of this file are subject to the Mozilla Public License
+## Version 1.1 (the "License"); you may not use this file except in
+## compliance with the License. You may obtain a copy of the License
+## at http://www.mozilla.org/MPL/
+##
+## Software distributed under the License is distributed on an "AS IS"
+## basis, WITHOUT WARRANTY OF ANY KIND, either express or implied. See
+## the License for the specific language governing rights and
+## limitations under the License.
+##
+## The Original Code is RabbitMQ.
+##
+## The Initial Developer of the Original Code is GoPivotal, Inc.
+## Copyright (c) 2007-2017 Pivotal Software, Inc.  All rights reserved.
+
+
+defmodule RabbitMQ.CLI.Ctl.Commands.QuorumStatusCommand do
+  require RabbitMQ.CLI.Ctl.InfoKeys
+  require RabbitMQ.CLI.Ctl.RpcStream
+
+  alias RabbitMQ.CLI.Ctl.RpcStream, as: RpcStream
+  alias RabbitMQ.CLI.Core.Helpers, as: Helpers
+
+  @behaviour RabbitMQ.CLI.CommandBehaviour
+  use RabbitMQ.CLI.DefaultOutput
+
+  def formatter(), do: RabbitMQ.CLI.Formatters.Erlang
+
+  def scopes(), do: [:ctl, :diagnostics]
+
+  defp default_opts() do
+    %{vhost: "/"}
+  end
+
+  def merge_defaults(args, opts), do: {args, Map.merge(default_opts(), opts)}
+
+  def validate([_|_] = args, _) when length(args) > 1 do
+    {:validation_failure, :too_many_args}
+  end
+  def validate([_], _), do: :ok
+  def validate([], _) do
+    {:validation_failure, :not_enough_args}
+  end
+
+  use RabbitMQ.CLI.Core.RequiresRabbitAppRunning
+
+  def run([name] = args, %{node: node_name, vhost: vhost}) do
+    :rabbit_misc.rpc_call(node_name, :rabbit_quorum_queue, :status, [vhost, name])
```

Comment:
There should be a better error message if trying to get status of a classic queue. Currently there is a case clause.